### DISCUSSION TITLE
Tyre Pressure Monitoring System

### DISCUSSION THREAD

**Discussion by Ciutnik** [devemux86/cruiser#587] [Show and tell]
Nice stuff. Admittedly very loosely related to navigation but strongly related to motorbikes (as well as bikes or cars) 🙂

After some recent adventures, I became interested in the Tyre Pressure Monitoring System (TPMS) on my motorbike. As I always have Android enabled with Cruiser, I decided that something that connects via BT directly to my phone/tablet would be best. I rejected special devices with a display that take up space on the handlebars, have to be charged and have to be guarded against being lost.

And so the search began. I guess there is no other option: either it is cheap (e.g. on AliExpress) or it is good. Since the matter is serious, I opted for a rather expensive (~$100) solution from Amazon: the highly praised FOBO Bike 2 from Malaysia. 

I can confirm that the device works sensationally! The readings are precise, they occur every half minute or so, the connection is stable. Three-level alerts are given when a drop in pressure, too low pressure and critically low pressure are detected. In addition, the software is able to convert the pressure of a warmed-up tyre to a value at 20°C (i.e. the recommended values submitted by manufacturers), as well as compensating the readings according to the barometric pressure at altitude! In the mountains this may be important. 

And now for the best: what is important for me, perhaps even decisive for choosing FOBO, the app interacts with Tasker! It only needs to listen to the intent `my.com.salutica.fobobike2.scan` to get the variables:
`%pressure`, `%temperature`, `%battery`, `%position`, `%lastreadtime`, `%macaddr`. Tasker can also enable or disable the background service that connects Android to sensors.

Simple automation allows me to achieve, for example, an effect like this (not continuous, of course!). For the time being, as a test, I have set myself up so that the pressures (and temperatures, battery levels, times) for the front and rear will appear to me briefly on start-up, and then when either pressure exceeds the alarm value - on a red background. Thanks to Tasker, I have absolute freedom here. But of course, without Tasker we will also receive alerts from the FOBO application!

![Screenshot_20240705_151312_Cruiser](https://github.com/devemux86/cruiser/assets/122647782/3c835cf2-1032-4b64-865a-f0961b584c54)

**Comment by devemux86** [comment]
@Ciutnik thank you for all this information, this is a very interesting read!
